src/pages/instances/TerminalPayloadForm.tsx

```diff
@@ -100,89 +112,72 @@ const TerminalPayloadForm: FC<Props> = ({ close, onFinish }) => {
           onBlur={formik.handleBlur}
           onChange={formik.handleChange}
           error={formik.touched.command ? formik.errors.command : null}
           value={formik.values.command}
-          stacked
         />
         <Input
           id="user"
           name="user"
-          label="User id"
+          label="User ID"
+          labelClassName="u-no-margin--bottom"
           type="number"
           onBlur={formik.handleBlur}
           onChange={formik.handleChange}
           value={formik.values.user}
-          stacked
         />
         <Input
           id="group"
           name="group"
-          label="Group id"
+          label="Group ID"
+          labelClassName="u-no-margin--bottom"
           type="number"
           onBlur={formik.handleBlur}
           onChange={formik.handleChange}
           value={formik.values.group}
-          stacked
         />
         {/* hidden submit to enable enter key in inputs */}
-        <input type="submit" hidden />
-        <Row>
-          <Col size={4}>
-            <Label>Environment variables</Label>
-          </Col>
-          <Col size={3}>Key</Col>
-          <Col size={3}>Value</Col>
-          <Col size={2} />
-        </Row>
-        {formik.values.environment.map((variable, index) => (
-          <Row key={index}>
-            <Col size={4} />
-            <Col size={3}>
-              <input
-                className="p-form-validation__input"
-                style={{ width: "10rem" }}
-                type="text"
-                name={`environment.${index}.key`}
-                onBlur={formik.handleBlur}
-                onChange={formik.handleChange}
-                value={formik.values.environment[index].key}
-              />
-            </Col>
-            <Col size={3}>
-              <input
-                className="p-form-validation__input"
-                style={{ width: "10rem" }}
-                type="text"
-                name={`environment.${index}.value`}
-                onBlur={formik.handleBlur}
-                onChange={formik.handleChange}
-                value={formik.values.environment[index].value}
-              />
-            </Col>
-            <Col size={2}>
-              <Button
-                onClick={() => removeEnvironmentRow(index)}
-                type="button"
-                hasIcon
-              >
-                <Icon name="delete" />
-              </Button>
-            </Col>
-          </Row>
-        ))}
-        <Row>
-          <Col size={4} />
-          <Col size={8}>
-            <Button onClick={addEnvironmentRow} type="button" hasIcon>
-              <Icon name="plus" />
-              <span>Add variable</span>
+        <Input type="submit" hidden />
+        <Label className="u-no-margin--bottom">Environment variables</Label>
+        {formik.values.environment.map((_variable, index) => (
+          <div key={index} className="env-variables">
+            <Input
+              type="text"
```

Comment:
This and the other input is missing a unique id. You can use the index to generate one.
We should also add a `label`, and the `labelClassName="u-off-screen"`.